config/jobs/kubernetes/sig-cloud-provider/azure/release-master.yaml

```diff
@@ -517,7 +517,8 @@ periodics:
   labels:
     preset-dind-enabled: "true"
     preset-kind-volume-mounts: "true"
-    preset-azure-cred: "true"
+    preset-azure-cred-only: "true"
+    preset-azure-anonymous-pull: "true"
```

Comment:
right now we're using a local registry for these jobs, do we need the external registry? Are we building k8s artifacts for these ones?

Reply:
If you just want to remove the SSH var you can use `preset-azure-cred-only` without `preset-azure-anonymous-pull`

Reply:
Switched back to local registry